ajax/libs/selectonic/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "selectonic",
+  "version": "0.6.3",
+  "description": "jQuery-plugin for making any list of items selectable by mouse and keyboard.",
+  "author": {
+    "name": "Alexey Novichkov",
+    "email": "mygrib@gmail.com"
+  },
+  "filename": "selectonic.min.js",
+  "keywords": [
+    "list",
+    "jquery",
+    "selectable",
+    "selection"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/anovi/selectonic.git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/anovi/selectonic.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      }
```

Comment:
Lets add another pattern in order to [fetch ](https://github.com/anovi/selectonic/tree/0.1.0)old versions
{
        "basePath": "",
        "files": [
          "jquery.multiSelectable*"
       ]
     }